secant/Screens/Onboarding/OnboardingFooterView.swift

```diff
@@ -13,61 +13,85 @@ struct OnboardingFooterView: View {
+                
+                if viewStore.isFinalStep {
+                    Button("Create New Wallet") {
+                        withAnimation(.easeInOut(duration: animationDuration)) {
+                            viewStore.send(.createNewWallet)
+                        }
+                    }
+                    .createButtonStyle
+                    .buttonLayout()
                     
-                    if viewStore.isFinalStep {
-                        Button("Create New Wallet") {
-                            withAnimation(.easeInOut(duration: animationDuration)) {
-                                viewStore.send(.createNewWallet)
-                            }
+                    Button("Import an Existing Wallet") {
+                        withAnimation(.easeInOut(duration: animationDuration)) {
+                            viewStore.send(.createNewWallet)
                         }
-                        .primaryButtonStyle
-                        .frame(height: proxy.size.height / 12)
-                        .padding(.horizontal, 15)
-                        .transition(.opacity)
-                    } else {
-                        Button("Next") {
-                            withAnimation(.easeInOut(duration: animationDuration)) {
-                                viewStore.send(.next)
-                            }
+                    }
+                    .secondaryButtonStyle
+                    .buttonLayout()
+                } else {
+                    Button("Next") {
+                        withAnimation(.easeInOut(duration: animationDuration)) {
+                            viewStore.send(.next)
                         }
-                        .primaryButtonStyle
-                        .frame(height: proxy.size.height / 12)
-                        .padding(.horizontal, 15)
-                        .transition(.opacity)
                     }
+                    .primaryButtonStyle
+                    .buttonLayout()
                     
                     ProgressView(
-                        "\(viewStore.index + 1)/\(viewStore.steps.count)",
+                        "0\(viewStore.index + 1)",
                         value: Double(viewStore.index + 1),
                         total: Double(viewStore.steps.count)
                     )
-                    .onboardingProgressStyle
-                    .padding(.horizontal, 30)
-                    .padding([.vertical], 20)
+                        .onboardingProgressStyle
+                        .padding(.horizontal, 30)
+                        .padding(.vertical, 20)
                 }
             }
         }
     }
 }
 
+struct OnboardingFooterButtonLayout: ViewModifier {
+    func body(content: Content) -> some View {
+        content
+            .frame(height: 60)
+            .padding(.horizontal, 28)
+            .transition(.opacity)
+    }
+}
+
+extension View {
+    func buttonLayout() -> some View {
```

Comment:
A global extension on view with a generic name like buttonLayout is a large hammer for something specific to Onboarding Footers.  What would you think about updating this name to be more specific or reducing the scope for easier understanding of the outcome or intent?